engine/src/main/java/org/terasology/engine/TerasologyEngine.java

```diff
@@ -155,6 +156,9 @@
      *                   audio and input subsystems.
      */
     public TerasologyEngine(TimeSubsystem timeSubsystem, Collection<EngineSubsystem> subsystems) {
+        // configure native paths
+        PathManager.getInstance();
```

Comment:
Why there?
What are means booleans?

Reply:
when the native paths are defined I also set the java paths. Just calling an instance of PathManager should be enough to set this up.